Dockerfile

```
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13 as builder
```

Shouldn't we use the latest Go version here?

```suggestion
FROM golang:1.15.7 as builder
```